src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,105 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size;
+    private int capacity;
+    private int threshold;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        capacity = DEFAULT_CAPACITY;
+        threshold = (int) (LOAD_FACTOR * capacity);
+        table = (Node<K, V>[]) new Node[capacity];
+    }
+
+    private void resize() {
```

Comment:
private methods should be at the end of the class, after public methods
or after public method where it is used
https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents

Reply:
done